packages/manager/modules/pci/src/components/project/flavors-list/translations/Messages_fr_FR.json

```
@@ -13,6 +13,10 @@
   "pci_project_flavors_spec_core_details": "{{vcores}} vCores ({{frequency}} GHz)",
   "pci_project_flavors_spec_disk": "Stockage :",
   "pci_project_flavors_spec_bandwidth": "Bande passante :",
+  "pci_project_flavors_spec_addons": "Addon :",
+  "pci_project_flavors_spec_addons_detail_single": "{{ addons }} card",
```

```suggestion
  "pci_project_flavors_spec_addons_detail_single": "{{ addons }} card",
```
I don't get what this is supposed to mean ?

---

Renamed the translation key to be more meaningful. It was supposed to be the number of GPU cards :)